snapcraft/internal/sources/__init__.py

```diff
@@ -169,20 +121,20 @@ def download(self):
         os.chmod(self.file, st.st_mode | stat.S_IEXEC)
 
 
-class Bazaar(Base):
+class Bazaar(_base.Base):
```

Comment:
Indeed, though a TODO isn't necessary-- once it's not needed, it'll be a static failure.